### PR TITLE
Doc changed

### PR DESCRIPTION
**Pull request by o0OooO**
*Issue #, if available:*

*Description of changes:*

*How Has This Been Tested:*

*[x] My testing has passed*

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: edge/cdk/extensions/prewarm/README.md
Comment (sussii) [inline edge/cdk/extensions/prewarm/README.md:1]: 1. Sign in to the Amazon Web Services Management Console and select the button to launch the CloudFormation template.
这句话不通。

参考：

1. Log in to the AWS Management Console and deploy the AWS CloudFormation stack using the follow link:
    * [Launch the stack in Global Regions] (https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=Prewarm&templateURL=https://aws-cloudfront-extensions-cff.s3.amazonaws.com/asset/prewarm/latest/PrewarmStack.template.json)

File: edge/cdk/extensions/prewarm/README-cn.md [hunk 1]
```diff
@@ -92,22 +122,32 @@ requestID: 预热请求id，您可通过调用PrewarmStatus API获取预热请
...
-status: 预热状态，COMPLETED表示完成，IN_PROGRESS表示进行中。
-total: 预热url的总数。
-completed: 预热完成的url的数量。
-inProgress: 正在预热中的url的数量。
+### 响应
+- 响应参数
+
+| **名称**     | **类型** | **描述**                             |
+|------------|--------|------------------------------------|
+| status     | *字符串*  | 预热状态，COMPLETED表示完成，IN_PROGRESS表示进行中 |
```
Comment (sussii) [inline edge/cdk/extensions/prewarm/README-cn.md:144]: 这里应该包含所有的values （英文同样的。）

File: docs-ig/zh/revisions.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
-| 2022年2月 | 发布版本1.3.0 <br>新增监控指标API |
-| 2022年8月 | 发布版本2.0.0 <br>新增web控制台 |
+| 日期       | 版本发布说明                                                                                                         |
+|----------|----------------------------------------------------------------------------------------------------------------|
+| 2021年3月  | 首次发布版本1.0.0                                                                                                    |
+| 2021年7月  | 发布版本1.1.0 <br> 1. 新增12个Lambda@Edge函数和9个CloudFront Functions<br>2. 新增基于Amazon WAF与Amazon Shield的CloudFront安全自动化 |
+| 2021年12月 | 发布版本1.2.0 <br>新增10个Lambda@Edge函数和2个CloudFront Functions                                                        |
+| 2022年2月  | 发布版本1.3.0 <br>新增监控指标API                                                                                        |
+| 2022年8月  | 发布版本2.0.0 <br>新增web控制台                                                                                         |
+| 2023年3月  | 发布版本2.1.0 <br>1. 优化预热扩展库，其中预热接口字段有调整，新增"protocol"和"target_type"字段，且"region"字段值有调整，具体请参考api文档<br>2. 新增监控指标      |
```
Comment (sussii) [inline docs-ig/zh/revisions.md:8]: 这个api文档部分 加一个链接。

File: docs-ig/en/revisions.md [hunk 3]
```diff
@@ -1,8 +1,9 @@
...
+| December 2021 | Released version 1.2.0  <br>1. Added 10 Lambda@Edge functions and 2 CloudFront functions                                                                                                                                                        |
+| February 2022 | Released version 1.3.0 <br>1. Added a CloudFront real-time log monitoring API which provides performance metrics                                                                                                                                |
+| August 2022 | Released version 2.0.0 <br>1. Added a user-friendly web console                                                                                                                                                                                 |
+| March 2023  | Released version 2.1.0 <br>1. Optimized the prewarm extension repository. The "protocol" and "target type" fields are added, and the value of the "region" field is adjusted. For details, see the api documentation<Br> New monitoring metrics |
```
Comment (sussii) [inline docs-ig/en/revisions.md:8]: add a document link

File: edge/cdk/extensions/prewarm/README.md [hunk 4]
```diff
@@ -51,43 +51,75 @@ To see details for the stack resources, choose the *Outputs* tab.
...
+      * target_type = "pop": this field accept a PoP list, pre-warm in the PoP location in the list,eg:["ATL56-C1", "DFW55-C3"]
+      * target_type = "region": this field  accept "all" or a region list, pre-warm in all regions or a specific region,eg:"all" or ["apac", "au"], the available regions are：
+        * apac： Asia-Pacific
+        * au： Australia
+        * ca： Canada
+        * sa： South Africa
+        * eu： Europe
+        * jp： Japan
+        * us： United States
+        * cn： China(Chinese mainland prewarm can only be used by deploying this solution in Chinese mainland regions, otherwise it will always fail.)
```
[the remark this answers is not in the document]
Comment (o0OooO) [reply edge/cdk/extensions/prewarm/README.md:82]: 之前版本是这样发布的， 看代码是包含了这几个pop 'IAD89-P1', 'IAD89-P2', 'SFO5-P2','SFO5-P1', 'DFW56-P1', 'DFW56-P2'
sufan要不直接确定下我们是否要修改成North America？

**Review by guming3d — DISMISSED** [review]
go shipping

**Pull request merged**